postgrest.conf

```diff
@@ -0,0 +1,4 @@
+db-uri = "postgres://postgres:mysecretpassword@localhost:5432/orderbook"
```

Comment:
It might be worth adding what you wrote in the commit message into the readme.md

Reply:
I might remove this file now since we have a docker container that does the same thing.

But maybe there is something to document in the Readme anyway.